crates/primitives/src/bits/macros.rs

```diff
@@ -65,6 +65,7 @@ macro_rules! wrap_fixed_bytes {
                 $extra_derives,
             )*
         )]
+        #[cfg_attr(feature = "diesel", derive(diesel_derive_newtype::DieselNewType))]
```

Comment:
this won't work if used outside this crate, please see how other traits are implemented below with a cfg(feature) macro_rules